app/auth.go

```diff
@@ -251,10 +275,6 @@ func (m *MenderAuthManager) Run() error {
 			m.fetchAuthToken(msg.ResponseChannel)
 		}
 	}
-	// stop the DBus interface, if available
-	if dbusConn != dbus.Handle(nil) && dbusLoop != dbus.MainLoop(nil) {
-		m.dbus.MainLoopQuit(dbusLoop)
-	}
```

Comment:
Only for my learning, we did you remove the MainLoopQuit here?

Reply:
I didn't remove it, I just moved it to a `defer` in the DBus initialization section above. It may or may not execute depending on the outcome of the initialization steps, so rechecking the pointers here is redundant.